arviz/plots/essplot.py

```diff
@@ -334,7 +334,7 @@ def plot_ess(
             ylabel.format("Relative ESS" if relative else "ESS"), fontsize=ax_labelsize, wrap=True
         )
         if kind == "evolution":
-            ax_.legend(title="Method", title_fontsize=xt_labelsize, fontsize=xt_labelsize)
+            ax_.legend(title="Method", fontsize=xt_labelsize)
```

Comment:
This was causing Sphinx to fail so I had to remove it :(

Reply:
Do you know why?

Reply:
Encountered unknown parameter. It is a bit strange.

Reply:
Reordering fixed it.